scss/bootstrap/_modal.scss

```diff
@@ -40,17 +40,19 @@
 .modal-dialog {
   position: relative;
   width: auto;
-  margin: $modal-dialog-margin;
+  margin: 10px;
```

Comment:
Should we be using the `$global-spacing-unit-tiny` here?

Reply:
I think `$global-spacing-unit-tiny` is 7.5px and `$global-spacing-unit-small` is 15px.. Hence I have replaced the value to be exactly as the Moodle's file dialog needed it.